src/main/scala/za/co/absa/spark/commons/common/Constants.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2018 ABSA Group Limited
```

Comment:
A lot of 2018 years in copyright. Please update to 2020 through the project

Reply:
Why 2020? Enceladus where the files comes from is from 2018. This library on the other hand from 2021,

Reply:
I am just forgetting what year it is